demos/linear-progress.html

```diff
@@ -16,7 +16,6 @@
     <meta charset="utf-8">
     <title>Linear Progress - Material Compoonents Catalog</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="assets/material-components-web.css.js" charset="utf-8"></script>
```

Comment:
Good catch. I fixed that in PR #1119, but forgot to backport it here.

Fixed it here - PTAL. Thanks!